README.md

```diff
@@ -75,10 +77,10 @@ Current methodological restrictions:
 
 Future releases will include:
 
-  - Support for parallelization over explanations, Monte Carlo sampling
+-   Support for parallelization over explanations, Monte Carlo sampling
     and features subsets for non-parallelizable prediction functions.
-  - Computational improvement of the AICc optimization approach
-  - Adaptive selection of method to account for the feature dependence
+-   Computational improvement of the AICc optimization approach
+-   Adaptive selection of method to account for the feature dependence
 
 Note that both the features and the prediction must be numeric. The
 approach is constructed for continuous features. Discrete features may
```

Comment:
may what?

Reply:
It's on the next line. You need to click the arrow-thing on the left side (right by the line numbers) to see more lines.